### DISCUSSION TITLE
Filtering on window.location.origin

### DISCUSSION THREAD

**Discussion by wake42** [plausible/analytics#2004] [Anything Else]
I see there is a `data-exclude` that lets you exclude certain pages from the sent statistics. I have the same site running in several environments with separate subdomains (www.domain.com,  www-test.domain.com, etc) is there some easy way of filtering on the domain the same was as `data-exclude` can filter on `windows.location`?

**Comment by RobertJoonas** [comment]
Hi @wake42!

Unfortunately we don't have anything specific built for this. The best option would be a domain specific site configuration - choose to use the snippet on production, but not on your testing domain. Another option is to keep the snippet there, but use the [localStorage flag](https://plausible.io/docs/excluding-localstorage) or [adblocker excluding](https://plausible.io/docs/excluding) to disable tracking the test domain on the devices you use. Hope this helps!

**Reply by wake42** [reply]
Thanks for your reply. I added a little script that checks what domain it is running on and set that localStorage flag for non prod environments. I guess there is a minor chance that the plausible script manages to send it's data before that flag is set, but that's good enough I think.
